e2etests/web/text_editing/build/web/assets/FontManifest.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
I think these manifest files should also be removed from version control.

Reply:
do you know if the app will run exactly the same without these? Especially assets.

Reply:
My understanding is that manifest files are auto-generated at build time. I could be wrong though.

Reply:
done. thanks!